packages/c/package.json

```diff
@@ -14,9 +14,7 @@
     "lib"
   ],
   "scripts": {
-    "start": "tsc -w",
```

Comment:
Do we need `tsconfig.json` in each package if we don't use tsc inside each one?

Reply:
Of course! We need every one of them because of the packages' tsconfig contains instructions on how to build this exact package. The root tsconfig has only references to projects we want to build. That's why the root config doesn't include any file - no `include`, empty `files: []`.